commands/beautify.ml

```diff
@@ -0,0 +1,27 @@
+open Api
+open Kernel
+open Cmdliner
+
+let beautify config files =
+  let open Processor in
+  let after _ = Option.iter (fun (env, lc, e) -> Env.fail_env_error env lc e) in
+  let hook = {before = ignore; after} in
+  Processor.handle_files files ~hook PrettyPrinter;
+  let f m =
+    let input = Parsers.Parser.input_from_stdin (Basic.mk_mident m) in
+    Processor.handle_input input PrettyPrinter
+  in
+  Option.iter f config.Config.run_on_stdin
+
+let files =
+  let doc = "Pretty print Dedukti file FILE" in
```

Comment:
Does it work for a single file or a list of files?

Reply:
Yes it should work for a list of files. I chose to write "file FILE" because the synopsis suggests that you can put several "FILE" (written as `[FILE]...`). So it process each `FILE` (which makes up a sequence of files).